docs/getting-started/community-r3f-components.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Community R3F Components
+description: This page showcases some React Three Fiber/r3f community components that have not been merged to the drei components collection or to another pmndrs project. 
+nav: 4
+---
+
+If you'd like to list new community components, please make a PR to [this doc](https://github.com/pmndrs/react-three-fiber/tree/master/docs/getting-started/community-r3f-components.mdx) file. It showcases some r3f community components that have not been merged to to the [drei](http://drei.docs.pmnd.rs) components collection or to another [pmndrs](https://github.com/pmndrs) project.
+
+## R3F community components 
+
+### Repos and docs
+
+This page showcases some React Three Fiber/r3f community components that have not been merged to the [drei](http://drei.docs.pmnd.rs) components collection or to another [pmndrs](https://github.com/pmndrs) project. If you'd like to list new community components, please make a PR to [this doc](https://github.com/pmndrs/react-three-fiber/tree/master/docs/getting-started/community-r3f-components.mdx) file. 
+
+#### Data sources
+
+ - [NASA-AMMOS/3DTilesRendererJS](https://github.com/NASA-AMMOS/3DTilesRendererJS/) (r3f [readme](https://github.com/NASA-AMMOS/3DTilesRendererJS/blob/master/src/r3f/README.md)) repo with doc
+ - Luma Labs [Gaussian Splats](https://cdn-luma.com/public/lumalabs.ai/luma-web-library/0.2/fefe154/index.html#react-three-fiber) renderer doc with demos
+ - [NYTimes/three-loader-3dtiles](https://nytimes.github.io/three-loader-3dtiles) 
+
+#### Renderers & frameworks
+
+ - Takram three-geospatial [clouds doc](https://github.com/takram-design-engineering/three-geospatial/tree/main/packages/clouds) and [demos](https://takram-design-engineering.github.io/three-geospatial/?path=/story/clouds-3d-tiles-renderer-integration--tokyo), and [atmosphere doc](https://github.com/takram-design-engineering/three-geospatial/tree/main/packages/atmosphere)
+ - [Looking Glass](https://docs.lookingglassfactory.com/developer-tools/webxr/react-three-fiber) doc and demos
+ - [Theatre-js](https://github.com/theatre-js/theatre) repo and [doc](https://www.theatrejs.com/docs/latest)
+
+#### Materials
+
+ - [FarazzShaikh/CustomShaderMaterial](https://github.com/FarazzShaikh/THREE-CustomShaderMaterial)
+ - [spite/THREE.MeshLine](https://github.com/spite/THREE.MeshLine)
```

Comment:
This is now https://github.com/pmndrs/meshline. Only Troika among these is maintained, though.

Reply:
Thanks for the feedback, just edited meshline repo and CSB. Do you want me to flag the projects that seem not maintained or leave them the way they are now?